core-libs/setup/recipes/b2b/config/default-b2b-occ-config.ts

```diff
@@ -1,17 +1,22 @@
+// We need this import for augmentation of OccEndpoints to pick up
+import { CheckoutOccEndpoints } from '@spartacus/checkout/occ';
 import { OccConfig } from '@spartacus/core';
 
+const defaultB2bCheckoutOccEndpoints: CheckoutOccEndpoints = {
+  setDeliveryAddress: 'orgUsers/${userId}/carts/${cartId}/addresses/delivery',
+  placeOrder: 'orgUsers/${userId}/orders?fields=FULL',
+};
+
```

Comment:
Would it have worked without defining `defaultB2bCheckoutOccEndpoints` ?

Reply:
It should. The reason I created this was more in common with maintenance of this file. We had to have import from the file where we augment the model and having an import without usage would probably be quickly removed by anyone using "organize import" feature. That's why I decided to split it and have reference to that model.